pipelines/common.yaml

```diff
@@ -70,6 +70,10 @@ spec:
       description: List of platforms to build the container images on. The available set of values is determined by the configuration of the multi-platform-controller.
       name: build-platforms
       type: array
+    - default: "false"
```

Comment:
Why this field change is nessary?

Reply:
update the description to add more information. basically, it is define to prefetch the dev managers, for example: fetch rpm packages defined in rpm lockfiles. refer to https://konflux.pages.redhat.com/docs/users/mintmaker/rpm-lockfile.html